examples/opup/test_demo.py

```diff
@@ -1,5 +1,17 @@
-from .main import demo
+import pytest
+
+from beaker import Application, sandbox
```

Comment:
sandbox used?

Reply:
dang I ran a solution wide check to make sure I hadn't missed anymore and was about to commit this one along with the previous but you were too quick 

fixed ✅